doc/Requirements Specification/Parts/Data.lyx

```diff
@@ -200,7 +200,21 @@ name "/D10/"
 \end_inset
 
  The software's source code.
- It must be written in Java and either
+ It must be written in Java 6 because 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SoMoX}
+\end_layout
+
+\end_inset
+
+ cannot cope with newer versions.
```

Comment:
That’s not the point here. Because SoMoX’ dependencies do not affect ours, as we don’t depend on SoMoX. It is, however, likely that we won’t support  > Java 6, because we’ll need JaMopp.

Let’s discuss about this, because I think we shouldn’t mention the analysed software’s Java version at all.